carma-messenger-config/chevrolet_tahoe_2018/docker-compose-background.yml

```diff
@@ -17,11 +17,11 @@ version: '2'
 
 services:
   carma-messenger-ui:
-    image: usdotfhwastol/carma-messenger-ui:3.1.0
```

Comment:
Hey Dan, we don't want to hardcode candidate or develop images into these files, but instead let docker hooks take care of this. The hooks i've added to this branch will take care of this. Please restore the image name to usdotfhwastol/carma-messenger-ui:3.1.0